cmd/vitconfig/vitconfig.go

```diff
@@ -50,9 +51,10 @@ type bftLeader struct {
 }
 
 type jcliProposal struct {
-	ExternalID string `json:"external_id"`
-	Options    uint8  `json:"options"`
-	Action     string `json:"action"`
+	ExternalID  string `json:"external_id"`
+	Options     uint8  `json:"options"`
+	Action      string `json:"action"`
+	Challengeid uint32 `json:"challenge_id"`
```

Comment:
nitpick: `ChallengeID` for consistent capitalization. Or should we be rusty and capitalize as `Id`?

Reply:
Yep, should be `ChallengeId` I think 😃